tailwind.config.ts

```diff
@@ -3,6 +3,12 @@ import defaultTheme from 'tailwindcss/defaultTheme'
 module.exports = {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
   theme: {
+    extend: {
```

Comment:
Please remove the entire extend block since we have no use for any of these utilities anymore :)